README.md

```diff
@@ -42,6 +45,28 @@ yarn create rwjblue-release-it-setup --labels-only
 npm init rwjblue-release-it-setup --labels-only
 ```
 
+### Changelog options
+
+If you want to configure your changelog to ignore merge commits, you can use the following option:
+
+```
+# in a yarn repo
+yarn create rwjblue-release-it-setup --no-changelog-merges
+
+# in an npm repo
+npm init rwjblue-release-it-setup --no-changelog-merges
+```
+
+If you want to configure your changelog to ignore dependabot commits, you can use the following option:
```

Comment:
I think we removed this option in the PR now?

Reply:
Correct. Will remove.